cypress/components/alert/alert.cy.tsx

```diff
@@ -101,13 +101,13 @@ context("Testing Alert component", () => {
     );
 
     it.each([
-      [SIZE.EXTRASMALL, 300],
-      [SIZE.SMALL, 380],
-      [SIZE.MEDIUMSMALL, 540],
-      [SIZE.MEDIUM, 750],
-      [SIZE.MEDIUMLARGE, 850],
-      [SIZE.LARGE, 960],
-      [SIZE.EXTRALARGE, 1080],
+      [SIZE.EXTRASMALL as string, 300],
+      [SIZE.SMALL as string, 380],
+      [SIZE.MEDIUMSMALL as string, 540],
+      [SIZE.MEDIUM as string, 750],
+      [SIZE.MEDIUMLARGE as string, 850],
+      [SIZE.LARGE as string, 960],
+      [SIZE.EXTRALARGE as string, 1080],
     ])(
```

Comment:
This should help with simplifying the type declaration a little bit, by instead writing the input as an array containing `[string, number]` tuples: 

```tsx
it.each([
      [SIZE.EXTRASMALL, 300],
      [SIZE.SMALL, 380],
      [SIZE.MEDIUMSMALL, 540],
      [SIZE.MEDIUM, 750],
      [SIZE.MEDIUMLARGE, 850],
      [SIZE.LARGE, 960],
      [SIZE.EXTRALARGE, 1080],
    ] as [string, number][])(
```